apps/worker/src/app/workflow/usecases/send-message/send-message-chat.usecase.ts

```diff
@@ -101,7 +102,10 @@ export class SendMessageChat extends SendMessageBase {
     } catch (e) {
       await this.sendErrorHandlebars(command.job, e.message);
 
-      return;
+      return {
+        status: 'failed',
+        detail: DetailEnum.MESSAGE_CONTENT_NOT_GENERATED,
```

Comment:
Maybe `reason` reads better than `detail` in singular?

Reply:
I am fine with both, moreover at the moment we do not use it, I added it "just in case"